frontend/src/pages/Skills/styled.ts

```diff
@@ -0,0 +1,58 @@
+import { styled, Grid, Box, Input, Divider } from '@mui/material';
+
+import theme from 'themeSettings';
+
+export const SkillsPageContainer = styled(Grid)({
+  [theme.breakpoints.down('xl')]: {
+    padding: '16px 24px 16px 14px',
+  },
+  [theme.breakpoints.up('xl')]: {
+    padding: '16px 30px 24px 0',
+    margin: 0,
+  },
+  maxWidth: '1482px',
+  paddingTop: '16px',
+});
+
+export const SearchWrapper = styled('div')({
+  marginLeft: 0,
+});
+
+export const SearchSkill = styled(Input)({
+  fontFamily: theme.typography.fontFamily,
+  fontWeight: '400',
+  fontSize: '26px',
+  lineHeight: '34px',
+  letterSpacing: '0.62px',
+  [theme.breakpoints.down('lg')]: {
+    fontSize: '20px',
+    lineHeight: '34px',
+  },
```

Comment:
lineHeight - add value at px bad practice. Default value in most of all 1.2 from fontSize. In your case 34/26 = 1.3
letterSpasing - you cant add px values like 0.1, 0.2. In default 2px. 

When copypasting from Figma, dont use all CSS props.

```suggestion
  fontWeight: 400,
  fontSize: '26px',
  lineHeight: 1.3,
  letterSpacing: '1px',
  [theme.breakpoints.down('lg')]: {
    fontSize: '20px',
  },
```

Reply:
![image](https://user-images.githubusercontent.com/44974457/164213525-e179f32e-bb8c-4ad0-9eed-e2f233acf773.png)
why we can't use 0.62px as in our Design? lets discuse about it